src/app/teams/join/[teamId]/page.tsx

```diff
@@ -0,0 +1,32 @@
+import {getServerAuthSession} from "~/server/auth";
+import {api} from "~/trpc/server";
+import JoinTeam from "~/app/_components/team/joinTeam";
+
+export default async function Home({ params }: { params: { teamId: string } }) {
+  const { teamId } = params;
+  const session = await getServerAuthSession();
```

Comment:
We should probably start capitalizing these before they get out of hand and cause bugs.

const { TEAM_ID } ...
const SESSION ...